src/components/PostPageDetail/index.js

```diff
@@ -55,7 +55,8 @@ const PostPageDetailIdComponent = (props) => {
     contextSource = CONTEXT_SOURCE.FEEDS,
     haveSeeMore,
     parentData,
-    isKeyboardOpen
+    isKeyboardOpen,
+    fromFeeds
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `fromFeeds` is added to the component's props. Ensure that this prop is always passed when using this component, or consider providing a default value for it to prevent potential issues with undefined values.
```diff
-    fromFeeds
+    fromFeeds = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->